lib/fs/basicfs_fileinfo_unix.go

```diff
@@ -8,7 +8,10 @@
 
 package fs
 
-import "syscall"
+import (
+	"os"
+	"syscall"
+)
```

Comment:
I don't think anything is changing in this file to require the new import.

Reply:
os.FileInfo is mentioned below.